### ISSUE TITLE
False positive from UnusedImports on provideDelegate operator functions

### ISSUE THREAD

**Issue by eskatos** [pinterest/ktlint#669]
The fix for #506 in https://github.com/pinterest/ktlint/commit/b997561532fffbc5c37b27e15aec9618c02c560d hardcoded the `org.gradle.kotlin.dsl.provideDelegate` import but `provideDelegate` operator functions aren't limited to `org.gradle.kotlin.dsl` and could be present in any Kotlin library.

See https://kotlinlang.org/docs/reference/delegated-properties.html#providing-a-delegate-since-11

The fix should be extended to any import of a `provideDelegate` operator function.

**Comment by JakeWharton** [comment]
I currently have this problem in [diffuse](https://github.com/JakeWharton/diffuse/) (git SHA ac5f1aa2c8cdcbb00beb0f3515201eb583385c77) from use of the [Clikt](https://github.com/ajalt/clikt) library:

https://github.com/JakeWharton/diffuse/blob/ac5f1aa2c8cdcbb00beb0f3515201eb583385c77/diffuse/src/main/kotlin/com/jakewharton/diffuse/diffuse.kt#L10

```
> Task :diffuse:ktlintMainSourceSetCheck FAILED
/Volumes/dev/JakeWharton/diffuse/diffuse/src/main/kotlin/com/jakewharton/diffuse/diffuse.kt:10:1: Unused import
```
